### DISCUSSION TITLE
Coins removed [] at height

### DISCUSSION THREAD

**Discussion by kissaev** [Chia-Network/chia-blockchain#5956] [Support]
![image](https://user-images.githubusercontent.com/5376567/119390720-1f5ace00-bcd6-11eb-8522-6750aeef3ab8.png)
What it is in logs? It's normal in latest version for linux?

**Comment by mariano54** [comment]
This is normal, you can ignore it. It means you did not spend any coins at that height.

**Reply by kissaev** [reply]
thanks for the answer

**Comment by BrandtH22** [comment]
@kissaev , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
